### PR TITLE
Create gradle project

### LINKED ISSUE #1: Create gradle project

### PR DESCRIPTION
**Pull request by lapter57**
Close #1 .

### CODE REVIEW

File: api/src/test/java/org/confeti/routers/StatusRouterTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    public void testStatusRoute() {
+        WebTestClient
+                .bindToRouterFunction(statusRoute)
+                .build()
+                .get().uri("/api/status")
+                .exchange()
+                .expectStatus().isOk()
+                .expectBody(String.class).isEqualTo(STATUS);
+    }
+}
```
Comment (marsofandrew) [inline api/src/test/java/org/confeti/routers/StatusRouterTest.java]: new line at the end of file

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    tasks.withType<JavaCompile> {
+        options.compilerArgs.addAll(listOf("--enable-preview", "-Xlint:preview"))
+    }
+    tasks.withType<Test> {
+        jvmArgs("--enable-preview")
+    }
+    tasks.withType<JavaExec> {
+        jvmArgs("--enable-preview")
+    }
+}
```
Comment (marsofandrew) [inline build.gradle.kts]: new line at the end of file

**Comment by marsofandrew** [comment]
Do we realy need this `gradle/wrapper/gradle-wrapper.jar` this binary file?

**Comment by lapter57** [comment]
> Do we realy need this `gradle/wrapper/gradle-wrapper.jar` this binary file?

Yes, it is important because we can build a project with Wrapper on any machine without the need to install Gradle first (details in [here](https://stackoverflow.com/questions/43954932/should-i-gitignore-my-gradlew) and [here](https://www.baeldung.com/gradle-wrapper))

**Review by marsofandrew — APPROVED** [review]

**Pull request merged**